ietf/api/urls.py

```diff
@@ -82,6 +82,8 @@
     url(r'^version/?$', api_views.version),
     # Application authentication API key
     url(r'^appauth/(?P<app>authortools|bibxml)$', api_views.app_auth),
+    # NFS metrics endpoint
+    url(r'^nfs_metrics/?$', api_views.nfs_metrics),
```

Comment:
bikeshed/overcomplication: maybe `metrics/nfs` to leave a path to other metrics?